### PR TITLE
docs: server setup config and documentation for librechat-echo

### PR DESCRIPTION
**Pull request by RiyaMathew-11**
This PR adds server setup configuration files and essential documentation for the LibreChat instance maintained by the org. Contains the following: 

- Contains script to clean db exports for usage report generation
- Server config setup and docker overrides for the instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,5 +8,5 @@ Shell scripts, ansible scripts and config files for foss united servers and self
 - [Nextcloud](https://nextcloud.com) - Experimenting with nextcloud for switching from the GSuite (like GDoc, GSpreadsheets)
 - [Vaultwarden](./servers/apps/vaultwarden/README.md) - Password Management on Organization and User level
 - [Mattermost](./servers/apps/mattermost/mattermost.md) - For Internal team discussions
-- [Librechat](https://www.librechat.ai/) - Custom ChatGPT Web UI called [Echo](https://echo.fossunited.org)
+- [Librechat](https://www.librechat.ai/) - AI chat platform/search engine called [Echo](https://echo.fossunited.org)
```
Comment (Mangeshrex) [inline README.md:11]: >/search engine 

I'm unsure in this part. Checked https://librechat.ai and it does not say that it is also a search engine ?

Comment (RiyaMathew-11) [reply README.md:11]: We are using it as a prompt based search engine too isnt it? I just mentioned the usecase we do for it.

File: servers/apps/librechat-echo/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## LibreChat
+
+LibreChat is a self-hosted solution designed for organization members to browse essential information using AI providers. The service is powered by the OpenAI API, with API usage facilitated through periodically purchased credits.
+
```
Comment (Mangeshrex) [inline servers/apps/librechat-echo/README.md:4]: could you please also mention what is the server name on linode ? With the specifications of the server ? CPU, Ram etc ? Just to keep things transparent and also new people would be able to understand that faster.

**Review by Mangeshrex — CHANGES_REQUESTED** [review]

File: servers/apps/librechat-echo/README.md (hunk 2 above)
Comment (RiyaMathew-11) [reply servers/apps/librechat-echo/README.md:4]: Added.

**Review by Mangeshrex — APPROVED** [review]

**Pull request merged**